canonical-web-and-design/charmed-kubeflow.io
# Build the dynamic content form

## Done
- Build the dynamic content form

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8046#get-in-touch
- Check the form content matches [the copy doc](https://docs.google.com/document/d/1PSLBJWNqWfonEfQFrWvJ4NpqCHcBoDR2wWmk8IldbOg/edit#)
- Check the form submits and goes to the live site. Change the live URL to demo keeping the path to see what you will see

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3166
